apps/scan/backend/src/app.ts

```diff
@@ -89,6 +89,20 @@ function buildApi(
       const [usbDrive] = await usb.getUsbDrives();
       assert(usbDrive?.mountPoint !== undefined, 'No USB drive mounted');
 
+      const authStatus = await auth.getAuthStatus(
+        constructAuthMachineState(workspace)
+      );
+      if (authStatus.status !== 'logged_in') {
```

Comment:
Might be worth a comment noting that, though the frontend tries to prevent this, someone may remove their card after this action is triggered, hence the check

Don't feel strongly though - once auth checks are packaged in a decorator, we won't have to repeat this comment over and over 😅

Reply:
[7404b8a](https://github.com/votingworks/vxsuite/pull/3184/commits/7404b8ae0889594031d5df3af8bc3d73c9a30cbc)